tests/_test_msui/test_mscolab.py

```diff
@@ -298,6 +298,14 @@ def setup(self, qtbot, mscolab_app, mscolab_server):
         # close all hanging operation option windows
         self.window.mscolab.close_external_windows()
 
+    def test_modify_mscolab_timeout(self, qtbot):
+        pytest.xfail("https://github.com/Open-MSS/MSS/issues/2716")
```

Comment:
This is different from using the xfail mark decorator in that it doesn't run the rest of the test. But the test should run so that we can catch if it starts to pass. The `strict` option for xfail also needs to be set so that we will notice when it starts to pass for some reason.

Reply:
thx for that hint and changed :)